dbcon/execplan/simplecolumn.h

```diff
@@ -57,8 +57,14 @@ class SimpleColumn : public ReturnedColumn
   /**
    * Constructors
    */
+  class ForTestPurposeWithoutOID{};
+
+
+
```

Comment:
Nit: It seems like CS keeps 2 blank lines between next function definition. Could you remove unnecessary blank lines.

Reply:
Done